distance3d/test/test_cube_unique_pairs_edge_case.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import unittest
+from distance3d import hydroelastic_contact
+import pytransform3d.rotations as pr
+
+
+def test_cube_unique_pairs(use_aabb_trees):
```

Comment:
this function shouldn't have test in the name, otherwise pytest runs it automatically and it fails because it got no argument